iqm5q/platform.py

```diff
@@ -152,10 +152,11 @@ def create():
 
     # flux
     for i in range(6, 11):
-        channels[f"L4-{i}"].power_range = 0.8
+        channels[f"L4-{i}"].power_range = 9
```

Comment:
This is not a valid value for power range for this instrument. This is HDAWG, and for it the power range seems to be in volts (while for the other instruments it is DB). The valid values are `200mV, 400mV, 600mV, 800mV, 1V, 2V, 3V, 4V, 5V`.

For testing my changes in https://github.com/qiboteam/qibolab/pull/848 I used 1V, and things worked perfectly fine.